### PR TITLE
 ✨Add HasLabels helper option for List calls

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

/assign @gerred @DirectXMan12

### CODE REVIEW

File: pkg/client/options.go [hunk 1]
```diff
@@ -388,6 +389,25 @@ func (m MatchingLabels) ApplyToDeleteAllOf(opts *DeleteAllOfOptions) {
 	m.ApplyToList(&opts.ListOptions)
 }
 
+// ContainsLabels filters the list/delete operation checking if the set of labels exists
+// without checking their values.
+type ContainsLabels []string
```
Comment (alvaroaleman) [inline pkg/client/options.go]: @vincepri What about `HasLabels`?  `Contains` to me sounds a bit like it was part of the spec or something

Comment (vincepri) [reply pkg/client/options.go]: sure that works too

Comment (vincepri) [reply pkg/client/options.go]: Fixed!

**Comment by ncdc** [comment]
@vincepri I'd suggest you retitle the PR based on the name adjustment

**Comment by vincepri** [comment]
@gerred ptal

**Comment by gerred** [comment]
@vincepri though seems like there's still a test issue

**Comment by vincepri** [comment]
It seems to be unrelated to these changes, seems like a flake?

/retest

**Comment by gerred** [comment]
yep. :+1:

**Pull request merged**